frontend/src/common/backend.ts

```diff
@@ -169,12 +169,32 @@ export type VersionDto = {
 export type AuditEventDto = {
   id: number;
   timestamp: Date;
-  type: 'CREATE_VAULT' | 'UNLOCK_VAULT' | 'UPDATE_VAULT_MEMBERSHIP';
+  type: 'CREATE_VAULT' | 'GRANT_VAULT_ACCESS' | 'REGISTER_DEVICE' | 'REMOVE_DEVICE' | 'UNLOCK_VAULT' | 'UPDATE_VAULT' | 'UPDATE_VAULT_MEMBERSHIP';
```

Comment:
With the steadily growing number of events, we should move related code to a separate file.

Maybe we can (later!) get rid of `backend.ts` altogether, cutting by domain, not by layer.